pkg/network/protocols/http2/protocol.go

```diff
@@ -115,16 +120,20 @@ func (p *protocol) ConfigureOptions(mgr *manager.Manager, opts *manager.Options)
 		MaxEntries: p.cfg.MaxTrackedConnections,
 		EditorFlag: manager.EditMaxEntries,
 	}
+
+	// Taking the max between fourth of p.cfg.MaxTrackedConnections to the default lower limit (1024), as it can
+	// introduce a major memory bump if we're using p.cfg.MaxTrackedConnections.
+	mapSize := uint32(math.Max(float64(p.cfg.MaxTrackedConnections)/4, mapSizeMinValue))
 	opts.MapSpecEditors[dynamicTable] = manager.MapSpecEditor{
-		MaxEntries: p.cfg.MaxTrackedConnections,
+		MaxEntries: mapSize,
```

Comment:
Why make it hard coded and not configurable so we can later change the value if needed?

Reply:
it is configurable, via `MaxTrackedConnections`